packages/airnode-deployer/src/handlers/aws/index.ts

```diff
@@ -139,6 +154,21 @@ export async function processHttpSignedDataRequest(
     format: parsedConfig.nodeSettings.logFormat,
     level: parsedConfig.nodeSettings.logLevel,
   });
+
+  // Check if the request origin header is allowed in the config
+  const originVerification = verifyRequestOrigin(
+    parsedConfig.nodeSettings.httpGateway.enabled ? parsedConfig.nodeSettings.httpGateway.corsOrigins : [],
+    event.headers['origin']
```

Comment:
Nit: Why not `.origin`? (We did not enable the TS setting for explicit property access so this syntax is not required).

Reply:
hmm no particular reason, I'll make the change